Core/src/org/sleuthkit/autopsy/actions/GetTagNameAndCommentDialog.java

```diff
@@ -128,8 +128,7 @@ private void display() {
 
             @Override
             public Component getListCellRendererComponent(JList<?> list, Object value, int index, boolean isSelected, boolean cellHasFocus) {
-                String status = ((TagName) value).getKnownStatus() == TskData.FileKnown.BAD ? TagsManager.getNotableTagLabel() : "";
-                String newValue = ((TagName) value).getDisplayName() + status;
+                String newValue = TagUtils.getDecoratedTagDisplayName((TagName) value);
```

Comment:
It looks like these are decorated right, but they are not ordered.  Is that a problem?

Reply:
The rendered is not involved in order.

Reply:
Sorry, I didn't have a better place to put the comment, but I don't this class is ordering the items that are are displayed in the drop down for the tag and comment dialog.  I'm thinking closer to 161 should probably be ordered if ordering needs to be applied to this list as well.  I'm also realizing that is probably more in reference to another PR where that was the issue at hand.  Sorry for the confusion on my part.

Reply:
Line 159, it gets the map and then puts it into a tree which puts the tags in alphabetical order.